README.markdown

```diff
@@ -100,6 +100,15 @@ Stick this in your vimrc:
 
 Note: Now start vim with plain `vim`, not `vim .`
 
+
+---
+> How can I open NERDTree as a tab window automatically when vim starts up on opening a directory?
```

Comment:
I don't like the term "tab window". Tabs and Windows are separate things in vim, and putting them together like this is a bit confusing. I suggest just "window".